Session/Conversations/ConversationVC.swift

```diff
@@ -1647,6 +1657,22 @@ final class ConversationVC: BaseVC, LibSessionRespondingViewController, Conversa
             completion?()
         }
     }
+    
+    private func makeInputViewFirstResponder() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) { [weak self] in
+            self?.inputAccessoryView?.becomeFirstResponder()
+        }
+    }
+    
+    func checkIfEventWasTriggerWhileNotVisible() -> Bool {
+        // Delegate reply action triggered by MessageInfoViewController
+        if let navigationStack = self.navigationController?.viewControllers {
+            if navigationStack.contains(where: { $0 is ConversationVC }) && navigationStack.last is MessageInfoViewController {
+                return true
+            }
+        }
+        return false
+    }
```

Comment:
Making this a function in a separate file makes the code more convoluted as it means someone trying to understand the code would need to jump to a different file - I've added another comment which should remove the need for this function anyway